app/keepers/keepers.go

```diff
@@ -249,7 +245,7 @@ func (ak *AppKeepers) InitKeepers(
 	// set memory store keys
 	// NOTE: The testingkey is just mounted for testing purposes. Actual applications should
```

Comment:
// NOTE: The testingkey is just mounted for testing purposes. Actual applications should

Should we remove this altogether?

Reply:
We could not sure what it's purpose is to be honest